engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -118,7 +118,9 @@
     private Vector3f relativeMovement = new Vector3f();
     private boolean isAutoMove = false;
     private boolean runPerDefault = true;
+    private boolean crouchPerDefault = false;
```

Comment:
Any reason to really have this one rather than just initialize `crouch` to `false` ?